installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_aws_rules.json

```diff
@@ -183,5 +183,6 @@
 	"aws_iamuser_shouldnothave_corenetwork_privileges",
 	"deny_public_access_to_rdp_Port_3389",
 	"enable_aws_config_service",
-	"aws_enable_encryption_for_rds_instance"
+	"aws_enable_encryption_for_rds_instance",
+	"integrate_cloudtrail_trails_with_cloudwatch_logs"
```

Comment:
couple days ago we had a chat with @plyubich and made a decision on policy naming.
It must follow next pattern: {source}_{asset_type}_{violation_name}
Please adjust it everywhere in you PR.

Reply:
Violation name should state what is wrong, not action required

Reply:
Addressed in commit